ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -285,6 +284,16 @@ public void shouldDeleteTopicThrowOnTopicDeletionDisabledException() {
     kafkaTopicClient.deleteTopics(Collections.singletonList(topicName2));
   }
 
+  public void shouldDeleteTopicsThrowKafkaDeleteTopicsException() {
+    // Given:
+    expect(adminClient.deleteTopics(anyObject())).andReturn(getKafkaDeleteTopicsException());
+    replay(adminClient);
+    final KafkaTopicClient kafkaTopicClient = new KafkaTopicClientImpl(adminClient);
+
+    // When:
+    kafkaTopicClient.deleteTopics(Collections.singletonList(topicName1));
```

Comment:
doesn't this throw?

Reply:
I had originally based this off a similar test in this folder, but then I realized that the other test was not in a very good state already. I fixed both of them now so they actually throw the expected errors.